CanHazFunny/CanHazFunny/IOutPutJoke.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny;
+
+public interface IOutPutJoke
+{
+    public void Print(String Joke);
```

Comment:
```suggestion
    public void Print(string Joke);
```
`string` preferred over `String`